tcrdd_test.sh

```diff
@@ -5,7 +5,7 @@ test_print_usage_when_asked() {
     headHash=$(runAsAlice getHeadHash)
     echo content > ${aliceClone}/aFile
     startStatus=$(runAsAlice git status -s)
-    runAsAlice ./tcrdd.sh -h true > $stdout
+    runAsAlice ./tcrdd.sh --help true > $stdout
```

Comment:
looks like the tests now use only the long verrsion, do you thing there is a way to tests both ? A kind of parameterized test using loops maybe ?

Reply:
How about rewriting the current tests as functions accepting the option flag as parameter and writing pairs of  tests that call the newly-created functions ?

Something akin to

```bash
should_print_usage_when_given() {
    option=$1
    # ...
    runAsAlice ./tcrdd.sh ${option} true > $stdout
    # ...
}

test_print_usage_when_given_short_option() {
    should_print_usage_when_given "-h"
}

test_print_usage_when_given_long_option() {
    should_print_usage_when_given "--help"
}
```

I think it would make the tests easier to read.